### PR TITLE
Audit logs user guide page

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Adds an audit logs page to the user guide Monitoring section that describes the Admin panel audit logs table and lists the application events logged.

### CODE REVIEW

File: platform_versioned_docs/version-23.4.0/administration/audit-logs.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- **Type**: A brief event description (`user_sign_in`, `credentials_created`, etc.) 
+- **Target**: ID of the resource associated with the event (ID of created credentials, etc.)
+- **Principal**: ID of the user that performed the action (user IDs for user-initiated events, `system` for system events)
+- **Status**: Additional event information, where applicable (workflow completion status, user sign-in method, etc.)
+- **Organization ID**
+- **Organization name**
+- **Workspace ID**
+- **Workspace name**
+- **Client IP**: IP address of user/client initiating the event
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-23.4.0/administration/audit-logs.mdx:22]:
```suggestion
- **Client IP**: IP address of user/client initiating the event (empty for system events)
```

File: platform_versioned_docs/version-23.4.0/monitoring/audit-logs.mdx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+description: An overview of application event audit logs in the Admin panel
+date: "08 Apr 2024"
+tags: [logging, audit logs, admin panel]
+---
+
+Root users can view application even audit logs from the [Admin panel](../administration/overview.mdx) **Audit logs** tab.
+
+## Audit log event format 
+
+Audit log entries record the following event details:
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.4.0/monitoring/audit-logs.mdx:12]: I might incorporate the parenthetical content into the sentences.

File: platform_versioned_docs/version-23.4.0/monitoring/audit-logs.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- **Type**: A brief event description (`user_sign_in`, `credentials_created`, etc.)
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.4.0/monitoring/audit-logs.mdx:14]:
```suggestion
- **Type**: A brief event description, such as `user_sign_in`, `credentials_created`, etc. 
```

File: platform_versioned_docs/version-23.4.0/monitoring/audit-logs.mdx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Root users can view application even audit logs from the [Admin panel](../administration/overview.mdx) **Audit logs** tab.
+
+## Audit log event format 
+
+Audit log entries record the following event details:
+
+- **Type**: A brief event description (`user_sign_in`, `credentials_created`, etc.) 
+- **Target**: ID of the resource associated with the event (ID of created credentials, etc.)
+- **Principal**: ID of the user that performed the action (user IDs for user-initiated events, `system` for system events)
```
Comment (jason-seqera) [inline platform_versioned_docs/version-23.4.0/monitoring/audit-logs.mdx:16]:
```suggestion
- **Principal**: ID of the user that performed the action for user-initiated events, or `system`, for Seqera Platform initiated events
```

**Review by jason-seqera — COMMENTED** [review]
Minor comments;

Google Developer style has a preference for avoiding frequently usage of parenthesis: https://developers.google.com/style/parentheses

**Review by jason-seqera — APPROVED** [review]
@llewellyn-sl, LGTM.

I'd probably `git rebase -i HEAD~11` and squash to a single commit for ease of reviewing concise git history. This can be helpful when going back and trying to see what commits are related to what overall issue while eliminating noise ("fix spacing", ect.). But that's a personal preference and we don't seem to do this as a general policy.

Thanks!

**Review by swampie — APPROVED** [review]

**Pull request merged**